lib/issues.go

```diff
@@ -51,14 +50,14 @@ func CompareIssues(config cfg.Config, ghClient clients.GitHubClient, jiraClient
 			id, _ := jIssue.Fields.Unknowns.Int(config.GetFieldKey(cfg.GitHubID))
 			if int64(*ghIssue.ID) == id {
 				found = true
-				if err := UpdateIssue(config, *ghIssue, jIssue, ghClient, jiraClient); err != nil {
+				if err := UpdateIssue(config, ghIssue, jIssue, ghClient, jiraClient); err != nil {
 					log.Errorf("Error updating issue %s. Error: %v", jIssue.Key, err)
```

Comment:
Precisely. Failing to create or update one issue shouldn't prevent us from creating the others, right? I think Ed specifically asked for that.